test/katex-spec.js

```diff
@@ -585,7 +585,7 @@ describe("An over/brace/brack parser", function() {
     it("should create a demonimator from the atoms after \\over", function() {
```

Comment:
Let's fix this typo at the same time:
```suggestion
     it("should create a denomimator from the atoms after \\over", function() {
```

Reply:
Right, I'll change it `denominator`.